web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
In general I think it is worth it look at this code:
```
function doSomething() {
    importSomeData();
    processTheData();
}

function importSomeData() {
    data = 'foo';
}

function processSomeData() {
     processedData = transform(data);
}

function transform (data) {
    return data * 2;
}
```
vs
```
var transform = function (data) {
    return data * 2;
}

var processSomeData = function() {
     processedData = transform(data);
}

var importSomeData = function () {
    data = 'foo';
}

var doSomething = function() {
    importSomeData();
    processTheData();
}
```

Which one is easier to follow?